src/config.rs

```diff
@@ -166,6 +167,26 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
+}
+
+impl<'a> Shell<'a> {
+    pub fn new(program: &'a str) -> Shell<'a> {
```

Comment:
Cool, we could use that for `new_with_args` that I think we need for cli parsing..